src/options.rs

```diff
@@ -518,7 +518,22 @@ fn conn_params(tmp: &ConnectionOptions) -> anyhow::Result<Builder> {
     let admin = tmp.admin;
     let user = env_fallback(tmp.user.clone(), "EDGEDB_USER")?;
     let host = env_fallback(tmp.host.clone(), "EDGEDB_HOST")?;
-    let port = env_fallback(tmp.port, "EDGEDB_PORT")?;
+    let port = match env_fallback(tmp.port, "EDGEDB_PORT") {
+        Ok(port) => port,
+        Err(e) => {
+            if let Ok(Some(val)) = get_env("EDGEDB_PORT") {
+                if val.starts_with("tcp://") {
+                    // This is likely set by Docker for linked container which
+                    // happens to have the name "edgedb"
+                    None
```

Comment:
Perhaps we should actually parse this instead of ignoring?

Reply:
I thought about that - maybe it makes more sense to parse it in the Docker image `edgedb/edgedb-cli`?

Reply:
Yeah, that's an even better idea.